modules/ROOT/pages/to-download-and-install-studio.adoc

```diff
@@ -9,4 +9,8 @@ To download and install Anypoint Studio, follow the instructions based on your o
 * xref:to-download-and-install-studio-ox.adoc[MacOS]
 * xref:to-download-and-install-studio-lx.adoc[Linux]
 
-Anypoint Studio 7.x only supports Mule 4.x projects since the structure of the project, export format, XML and scripting language are different.
\ No newline at end of file
+Anypoint Studio 7.x only supports Mule 4.x projects since the structure of the project, export format, XML and scripting language are different.
+
+== Update Schedule
+
+Anypoint Studio 7.15+ and Mule runtime engine 4.5+ are fully aligned in their release cadence. 
```

Comment:
Mule 4.5 release is happening on March 7th. There is no 4.5 content accessible for customers yet. I would wait till that release to merge this PR.